### ISSUE TITLE
'js-ObjTree' is not in the npm registry

### ISSUE THREAD

**Issue by makc** [rranauro/ObjTree#3]
when I try to install using npm, I get this:
```
npm WARN package.json jox@0.0.1 No repository field.
npm http GET https://registry.npmjs.org/js-ObjTree
npm http 404 https://registry.npmjs.org/js-ObjTree
npm ERR! 404 'js-ObjTree' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, or http url, or git url.

npm ERR! System Darwin 12.5.0
npm ERR! command "/usr/local/bin/node" "/usr/local/bin/npm" "install" "js-ObjTree"
npm ERR! cwd /Users/makc/Downloads/threejs.org/examples/С бетонной частью/1
npm ERR! node -v v0.10.26
npm ERR! npm -v 1.4.3
npm ERR! code E404
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/makc/Downloads/threejs.org/examples/С бетонной частью/1/npm-debug.log
npm ERR! not ok code 0
macbookpro-00254ba7af02:1 makc$ npm install js-ObjTree
npm WARN package.json jox@0.0.1 No repository field.
npm http GET https://registry.npmjs.org/js-ObjTree
npm http 404 https://registry.npmjs.org/js-ObjTree
npm ERR! 404 'js-ObjTree' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, or http url, or git url.

npm ERR! System Darwin 12.5.0
npm ERR! command "/usr/local/bin/node" "/usr/local/bin/npm" "install" "js-ObjTree"
npm ERR! cwd /Users/makc/Downloads/threejs.org/examples/С бетонной частью/1
npm ERR! node -v v0.10.26
npm ERR! npm -v 1.4.3
npm ERR! code E404
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/makc/Downloads/threejs.org/examples/С бетонной частью/1/npm-debug.log
npm ERR! not ok code 0
```

**Comment by jsfrocha** [comment]
@makc : `npm install objtree` worked for me..

**Comment by makc** [comment]
that's probably because it was fixed in commit 1410c21
